tests/podcast.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect, } from '@playwright/test';
+import { copyWithButton, } from './helper';
+
+test('Choose any podcast. Verify the shareable link matches the text copied from the Copy Link button' , async ({ page, context }) => {
+
+    await page.goto('https://music.amazon.com/')
+    page.waitForLoadState('domcontentloaded')
+    // click on podcasts
+    await page.getByRole('link', { name: 'Podcasts' }).click();
+    await expect(page).toHaveURL('https://music.amazon.com/podcasts')
+
+    // click on the first podcast shown
+    await page.locator('music-vertical-item').first().click();
+    await expect(page).toHaveURL(/https:\/\/music\.amazon\.com\/podcasts\/.+/)
+
+    //click the share button
+    const shareButton = page.getByTestId('detailHeaderButton2')
+    await shareButton.click();
+
+    const linkField = await page.locator('._3QpxCCZ2ZUyhnlmpwSU7as ').inputValue()
```

Comment:
This also works:

`const linkField = await page.getByTestId("dialog").locator("input").inputValue();`

## The parent div has an `id=dialog`
![image](https://github.com/user-attachments/assets/54853bf7-63f9-412d-b6ae-0639d057f8df)

## Using the locator in a debug sesh
![image](https://github.com/user-attachments/assets/8e70620b-52e7-4036-8b99-7bfb4f4036e7)

Reply:
I feel 100000000% better about this locator! Good find, my eyes overlooked this ID. 

Also I think I need a lesson on using the debugger.. maybe I'll find some youtube videos.... unless you teach it in the course? I remember the section on the trace viewer, but not the debugger.